python/res/fm/templating/__init__.py

```diff
@@ -0,0 +1,5 @@
+from .template_render import render_template
+from .template_render import load_data
+
+def load_parameters():
+    return load_data('parameters.json')
```

Comment:
Please add a new line to the files with the no-newline symbol displayed here at _Github_ ;)